PhysicsTools/PatAlgos/test/miniAOD/patTuple_mini_data.py

```diff
@@ -26,7 +27,12 @@
 process.load("SimGeneral.HepPDTESSource.pythiapdt_cfi")
 
 process.load("PhysicsTools.PatAlgos.slimming.slimming_cff")
+patAlgosToolsTask.add(process.slimmingTask)
+
 process.load("RecoVertex.AdaptiveVertexFinder.inclusiveVertexing_cff")
+patAlgosToolsTask.add(process.inclusiveVertexingTask)
```

Comment:
one load, one task add could be a more clean implementation pattern.
Better yet, if this can be done in one line with a "smarter" method .. although this will probably not always work